test/config-next/ca.json

```diff
@@ -63,35 +63,35 @@
 				{
 					"useForRSALeaves": false,
 					"useForECDSALeaves": true,
-					"issuerURL": "http://127.0.0.1:4001/aia/issuer/5214744660557630",
+					"issuerURL": "http://127.0.0.1:4502/int ecdsa a",
```

Comment:
Yep, that's correct! It doesn't _have_ to be, we can do this however we want (use hyphens instead of spaces, whatever) but I wrote aia-test-srv to correctly unquote spaces:
https://github.com/letsencrypt/boulder/blob/95db4614356d0c0a7d860a78923b91cd7a827493/test/aia-test-srv/main.go#L23

If you'd prefer something that doesn't look quite as odd, I'm happy to change it. Got ideas?

Reply:
Aha, I didn't go investigate there. It's fine to me as-is knowing this.